sql/plan/ddl_trigger.go

```diff
@@ -81,7 +81,8 @@ func (c *CreateTrigger) WithDatabase(database sql.Database) (sql.Node, error) {
 }
 
 func (c *CreateTrigger) Resolved() bool {
-	return c.ddlNode.Resolved() && c.Table.Resolved() && c.Body.Resolved()
+	// c.Body can be unresolved since it can have unresolved table reference to non-existent table
```

Comment:
I think this makes sense and seems to be the same behavior as MySQL (not fully resolving the trigger body). I couldn't find a reference in the docs that described how MySQL handles the trigger block, but I tested a couple other cases, such as unresolved/non-existent functions and confirmed MySQL will happily create the trigger with unknown functions and only error when it's used. 

On a related note... stored procedures seem like they need the same handling. They have the same behavior in MySQL where functions/tables aren't fully resolved until the procedure is executed. You don't need to add that into this PR, but as you're looking through this code, it might be good to think about similarities with StoredProcedures. At minimum we should create a tracking issue for it (probably label it with `sql-coverage` or maybe it's even more correct to label as `mysql-behavior` or something), but it'd be even better to knock it out after you complete this one and still have all the code fresh in your mind.